yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/command/QueryConfig.java

```diff
@@ -342,6 +345,19 @@ void checkResultInternal(@Nonnull Object actual, @Nonnull String queryDescriptio
         };
     }
 
+    private static QueryConfig getSupportedVersionConfig(final Object value, final int lineNumber, final YamlExecutionContext executionContext) {
+        final SupportedVersionCheck check = SupportedVersionCheck.parse(value, executionContext);
+        if (!check.isSupported()) {
+            return new SkipConfig(QUERY_CONFIG_SUPPORTED_VERSION, value, lineNumber, executionContext, check.getMessage());
+        }
+        return new QueryConfig(QUERY_CONFIG_SUPPORTED_VERSION, value, lineNumber, executionContext) {
+            @Override
+            void checkResultInternal(@Nonnull final Object actual, @Nonnull final String queryDescription) throws SQLException {
+                // Nothing to do, this query is supported
```

Comment:
This should probably be through a `RuntimeException` - which could ensure that this config is never run for a query. It's mostly precautionary.

Reply:
Done.